### PR TITLE
Python 3.8 fixes

### PR DESCRIPTION
**Pull request by Kodiologist**

### CODE REVIEW

**Comment by Kodiologist** [comment]
I'm not sure what to make of the test failures on Travis. `nosetests --nocapture` passes on my own machine (running Ubuntu 18.10) with Python 3.8.0a2.

**Comment by berkerpeksag** [comment]
I guess the cause of the failure is that something has changed in the development version of Python. Also, our Travis configuration seems wrong, because the following lines were supposed to make the build for 3.8-dev passed in case of a failure:

https://github.com/berkerpeksag/astor/blob/641390339bea88f13bcdd0abaa576e078191a86c/.travis.yml#L20-L21

**Comment by Kodiologist** [comment]
I removed those lines in this PR, because I expected the build to pass. And I think I'm using the same version of Python 3.8.

**Comment by Kodiologist** [comment]
@berkerpeksag Okay, all should be good now.

File: docs/changelog.rst [hunk 1]
```diff
@@ -29,6 +29,16 @@ Bug fixes
...
 
+* Fixed code generation with attributes of integer literals.
+  (Fixed by Kodi Arfer in `PR 133`_.)
+
+.. _`PR 133`: https://github.com/berkerpeksag/astor/pull/133
+
+* Fixed code generation with ``u``-prefixed string literals.
+  (Fixed by Kodi Arfer in `PR 133`_.)
+
+.. _`PR 133`: https://github.com/berkerpeksag/astor/pull/133
```
Comment (berkerpeksag) [inline docs/changelog.rst]: I think we can reuse the first one here.

**Comment by berkerpeksag** [comment]
Also, I usually use the "squash and merge" option, but let me know if you'd like to keep these commits separated.

File: docs/changelog.rst (hunk 1 above)
Comment (Kodiologist) [reply docs/changelog.rst]: Done.

**Comment by Kodiologist** [comment]
> feel free to submit a separate PR (or use this one if you prefer) to add Python 3.8 to trove classifiers and update other places (I can't remember if there's more places we need to update :))

I've updated the trove classifiers, but I can't remember anything else that needs to be changed, either.

> Also, I usually use the "squash and merge" option, but let me know if you'd like to keep these commits separated.

It's more your project than mine, but my philosophy has always been that it's easier to read history in bite-sized chunks.

**Comment by berkerpeksag** [comment]
Thank you!

**Pull request merged**